### ISSUE TITLE
Accessibility checker is not working correctly

### ISSUE THREAD

**Issue by AlexanderMelox** [carbon-design-system/ibm-products#2154]
## What will this achieve?
Currently, some tests with potential violations are failing when they shouldn't. 

We have an accessibility checker config stating to only fail on violations but clearly this isn't the case when running the test suites.

`ibm-cloud-cognitive/packages/core/accessibility-checker/index.js`

```js
module.exports = {
  failLevels: ['violation'],
  policies: ['WCAG_2_1'],
};

```

To try to fix it @matthewgallo and I tried moving the `achecker.js` file to the root, since the #accessibility-at-ibm channel instructed us to.

<img width="1317" alt="Screen Shot 2022-08-04 at 1 35 16 PM" src="https://user-images.githubusercontent.com/12755042/182914744-c7965f48-fea7-4ca7-92bd-9864b422d4ed.png">

When doing this, the potential violations went away, but now a bunch of component suites are failing their a11y checks

<img width="664" alt="Screen Shot 2022-08-04 at 1 36 05 PM" src="https://user-images.githubusercontent.com/12755042/182914893-5ad6facd-ceb3-446d-a032-71e57257445c.png">

**Comment by elycheea** [comment]
Will need to continue to look into this.
